docs/opentracing.rst

```diff
@@ -0,0 +1,179 @@
+===========
+Opentracing
```

Comment:
This seems to be spelt `OpenTracing` in the upstream docs. Please can you use that consistently, rather than "Opentracing" / "opentracing"?

Reply:
(likewise `synapse` -> `Synapse`, while I'm on naming)